tsconfig.json

```diff
@@ -1,120 +1,130 @@
 {
```

Comment:
Cleanup the `tsconfig.json`, remove unused option comments or extend a reasonable config.

Reply:
If you want to get rid of all the commented things I am fine with that